src/core_plugins/metrics/public/components/vis_types/table/vis.js

```diff
@@ -176,10 +184,13 @@ class TableVis extends Component {
     if (_.isArray(visData.series) && visData.series.length) {
       rows = visData.series.map(this.renderRow);
     } else {
-      let message = 'No results available.';
-      if (!model.pivot_id) {
-        message += ' You must choose a group by field for this visualization.';
-      }
+      const message = (<FormattedMessage
+        id="tsvb.table.noResultsAvailableMessage"
+        defaultMessage="No results available.{chooseGroupByField}"
+        values={{
+          chooseGroupByField: !model.pivot_id ? ' You must choose a group by field for this visualization.' : ''
```

Comment:
you did not translate this message. also please create two separate messages. One for case when model.pivot_id is false and one when model.pivot_id is true